index.js

```diff
@@ -28,7 +28,7 @@ function *autodiscover() {
 		null;
 	const xmlns		= schema !== null ? 
 		schema.content : 
-		"http://schemas.microsoft.com/exchange/autodiscover/responseschema/2006";
+		"http://schemas.microsoft.com/exchange/autodiscover/outlook/responseschema/2006a";
```

Comment:
That does not seem right: the "a" at the end looks like a typo, and I have not seen that schema in Microsoft doc.
```suggestion
		"http://schemas.microsoft.com/exchange/autodiscover/outlook/responseschema/2006";
```